topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>    >
+>    > * If you are sure the format is compatible with a tabular datatype, as this is the case here ;), you can manually change it, clicking on the eye of the "wolf_diet_ngsfilter.txt" dataset, then selecting the "Datatypes" section then affecting manually tabular and saving the operation
+>    >
+>    {: .tip}
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > Each sequence record is assigned to its corresponding sample and marker using the data provided in a text file (here wolf_diet_ngsfilter.txt). This text file contains one line per sample, with the name of the experiment (several experiments can be included in the same file), the name of the tags (for example: aattaac if the same tag has been used on each extremity of the PCR products, or aattaac:gaagtag if the tags were different), the sequence of the forward primer, the sequence of the reverse primer, the letter T or F for sample identification using the forward primer and tag only or using both primers and both tags, respectively.
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many sequences are not assigned?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. 1391
+> >
+> {: .solution}
+>
+{: .question}
+
+## Dereplicate reads into uniq sequences with **obiuniq**
+
+> ### {% icon hands_on %} Hands-on: Groups together sequence records
+>
+> 1. {% tool [obiuniq](toolshed.g2.bx.psu.edu/repos/iuc/obi_uniq/obi_uniq/1.2.13) %} with the following parameters:
+>    - *"Input sequences file"*: `Trimmed and annotated file by NGSfilter`
+>    
+>    - *"Attribute to merge"*: `sample`
+>    
+>    - *"Use specific option"*: `merge`
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > The same DNA molecule can be sequenced several times. In order to reduce both file size and computations time, and to get easier interpretable results, it is convenient to work with unique sequences instead of reads. To dereplicate such reads into unique sequences, we use the obiuniq command.
+>    > Definition: Dereplicate reads into unique sequences
+>    > * compare all the reads in a data set to each other
+>    > * group strictly identical reads together
+>    > * output the sequence for each group and its count in the original dataset (in this way, all duplicated reads are removed)
+>    > *Definition adapted from Seguritan and Rohwer (2001)*
+>    {: .comment}
+>
+{: .hands_on}
+
+> ### {% icon question %} Questions
+>
+> 1. How many sequences you had and how many you finally obtain?
+>
+> > ### {% icon solution %} Solution
+> >
+> > 1. From 43 326 to 3 962
+> >
+> {: .solution}
+>
+{: .question}
+
+
+## Limit number of informations with **obiannotate**
+
+> ### {% icon hands_on %} Hands-on: Adds/Edits sequence record annotations
+>
+> 1. {% tool [obiannotate](toolshed.g2.bx.psu.edu/repos/iuc/obi_annotate/obi_annotate/1.2.13) %} with the following parameters:
+>    - In *"Keep only attribute with key"*:
```

Comment:
Which input file should we use in this step?

Reply:
added "*"Input sequences file"*: `obiuniq output file`"

Reply:
THANK YOU!